test/tests/controls/pid_control/tests

```diff
@@ -8,8 +8,19 @@
       input = 'pid_control.i'
       csvdiff = 'out.csv'
       recover = false
+      # Not ouputting left_boundary_average to avoid regold
+      cli_args = 'Postprocessors/left_boundary_average/outputs=none'
       detail = 'using the principle of Proportional Integral Derivative control,'
     []
+    [min_max_limits]
+      type = 'CSVDiff'
+      input = 'pid_control.i'
+      csvdiff = 'min_max_limits.csv'
+      recover = false
+      # Not ouputting left_boundary_average to avoid regold
```

Comment:
```suggestion
```

Reply:
removed the comment in question and fixed the typo in the other two comments I added